docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: GPL-2.0
+
+# Copyright (C) 2024 Randolph Sapp
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, see <https://www.gnu.org/licenses/>.
+
+FROM debian:stable-slim
```

Comment:
Also add:
LABEL org.opencontainers.image.source https://github.com/TexasInstruments/processor-sdk-doc

Reply:
This label is added automatically by the workflow metadata script. Along with a few other useful labels.